src/test/no_pmix_launcher_client.c

```diff
@@ -178,6 +254,17 @@ int main(int argc, char **argv)
 		assert(0);
 	}
 
+	printf("\nSCHAN15 - Sync'ing all ep\n\n");
+	//for (tag = 0; tag < NUM_SERVER_CTX; tag++) {
+		rc = wait_for_ranks(crt_ctx, grp, rank_list, NUM_SERVER_CTX - 1, 3);
+        	if (rc != 0) {
+                	D_ERROR("wait_for_ranks() failed; rc=%d\n", rc);
+                	assert(0);
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs